src/Bundle.js

```diff
@@ -161,6 +161,7 @@ export default class Bundle {
 			return Promise.resolve( this.resolveId( source, module.id ) )
 				.then( resolvedId => {
 					if ( !resolvedId ) {
+						if ( source[0] === '.' ) throw new Error( `Could not resolve ${source} from ${module.id}` );
```

Comment:
Technically this should be starting with `./` or `../` since you could have a file or package starting with a period.